src/components/shared/QuestionHeader.jsx

```diff
@@ -43,34 +43,6 @@ class QuestionHeader extends React.Component {
   componentDidMount() {
     this.syncPropsAndState(this.props);
   }
-  shouldComponentUpdate(newProps, newState) {
```

Comment:
Why did you remove this?

Reply:
I took it out and nothing about the application changed. The component was not rerendering more than necessary.